web/src/enterprise/repo/settings/RepoSettingsCodeIntelligencePage.tsx

```diff
@@ -1,36 +1,29 @@
 import * as GQL from '../../../../../shared/src/graphql/schema'
-import React, { FunctionComponent, useCallback, useEffect, useMemo } from 'react'
+import React, { FunctionComponent, useCallback, useEffect, useState } from 'react'
 import { eventLogger } from '../../../tracking/eventLogger'
-import { fetchLsifDumps, fetchLsifUploads } from './backend'
 import { FilteredConnection, FilteredConnectionQueryArgs } from '../../../components/FilteredConnection'
 import { Link } from '../../../../../shared/src/components/Link'
 import { PageTitle } from '../../../components/PageTitle'
 import { RouteComponentProps } from 'react-router'
 import { Timestamp } from '../../../components/time/Timestamp'
 import { Collapsible } from '../../../components/Collapsible'
-import { useObservable } from '../../../util/useObservable'
-import { Observable } from 'rxjs'
+import { fetchLsifUploads } from './backend'
+import { Toggle } from '../../../../../shared/src/components/Toggle'
 
-const LsifDumpNode: FunctionComponent<{ node: GQL.ILSIFDump }> = ({ node }) => (
-    <div className="w-100 list-group-item py-2 lsif-data__main">
-        <div className="lsif-data__meta">
-            <div className="lsif-data__meta-root">
-                <code>{node.projectRoot?.commit.abbreviatedOID || node.inputCommit.substring(0, 7)}</code>
-                <span className="ml-2">
-                    {node.projectRoot ? (
-                        <Link to={node.projectRoot.url}>
-                            <strong>{node.projectRoot.path || '/'}</strong>
-                        </Link>
-                    ) : (
-                        node.inputRoot || '/'
-                    )}
-                </span>
-            </div>
-        </div>
+interface ToggleComponentProps {
+    onlyCompleted: boolean
+    onToggle: (enabled: boolean) => void
+}
 
-        <small className="text-muted lsif-data__meta-timestamp">
-            <Timestamp noAbout={true} date={node.processedAt} />
-        </small>
+const ToggleComponent: FunctionComponent<ToggleComponentProps> = ({ onlyCompleted, onToggle }) => (
```

Comment:
```suggestion
const HideIncompleteLSIFUploadsToggle: FunctionComponent<ToggleComponentProps> = ({ onlyCompleted, onToggle }) => (
```
Removing the Hungarian notation and giving the component a more precise name (what is toggling?). The CSS class name should be updated accordingly (`hide-incomplete-lsif-uploads-toggle`)